### DISCUSSION TITLE
turn off email auth

### DISCUSSION THREAD

**Discussion by dectomroc** [supabase/supabase#29015] [Questions] [answered]
Is the a non-smtp way of completely turning off email authentication upon user signup?

I was able to turn it off a couple weeks ago, but I went to rebuild my project from the ground up and it now seems every new user I try signup up for the sake of testing has to go through an email verification, as I can't find that same button that let my simply turn it off

**Comment by GaryAustin1** [comment, marked as answer]
Maybe you mean this?
![image](https://github.com/user-attachments/assets/d442b5b5-df0c-4ce9-8f48-2cc216e7d5f0)

**Reply by dectomroc** [reply]
my bad
